Wikipedia/Code/WMFArticleContainerViewController+TOC.swift

```diff
@@ -50,7 +50,11 @@ extension WMFArticleViewController : WMFTableOfContentsViewControllerDelegate {
     }
 
     public func tableOfContentsArticleSite() -> MWKSite {
-        return self.articleTitle.site
+        if(self.articleTitle.isNonStandardTitle()){
+            return MWKSite.init(language: "en")
```

Comment:
Interesting... so it's not that we're using localized strings so we can't rely on that english fallback.

So for now we assume all non-standard TOC sites are English... which works because at the moment only the FAQ uses the TOC in this way?